runtime/lua/vim/treesitter/query.lua

```diff
@@ -99,9 +99,23 @@ end
 
```

Comment:
a the moment `get_query_files` does not regard `explicit_query_files` but it probably should